src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -369,7 +384,7 @@ ManifestCache::load (
             beast::detail::base64_decode(configManifest));
         if (! mo)
         {
-            JLOG (j_.error()) << "Malformed manifest in config";
+            JLOG (j_.error()) << "Malformed token manifest in config";
```

Comment:
Do you mean to call this a "token manifest"?  Perhaps you meant "manifest token"?  Would "manifest blob" be clearer?